ml-agents/mlagents/trainers/buffer.py

```diff
@@ -240,6 +282,22 @@ def reset_update_buffer(self):
         """
         self.update_buffer.reset_agent()
 
+    def truncate_update_buffer(self, max_length):
+        """
+        Truncates the update buffer to a certain length.
+
+        The skew makes a bigger truncation each time, hopefully requiring less truncations overall.
```

Comment:
What is the skew?

Reply:
Seems like an old comment from an older implementation - removed.